_reports/examples/llm-visibility-toolbox/report.html

```
@@ -1668,28 +2488,77 @@ <h2 class="no-chapter" id="executive-summary-badge-rct">Executive summary <span
 <p>V4 adds schema-downgrade rationale, engine-specific reporting, source grouping, and diagram/equation fallbacks.</p>
 </details>
 <section class="action-line">
 <p><strong>Operator action:</strong> collect source IDs first, then interpret findings into recommendations with owners, acceptance criteria, and rerun steps.</p>
-</section><details class="accordion action-prompt"><summary>Action Prompt</summary><div class="code-block-wrap"><div class="code-block-head"><span>Copyable action prompt</span><button class="code-copy" type="button" aria-label="Copy code" title="Copy code">⧉</button></div><pre class="language-text"><code>Reference this report action: collect source IDs first, then interpret findings into recommendations with owners, acceptance criteria, and rerun steps.
+</section><details class="accordion action-prompt" open><summary>Action Prompt</summary><div class="code-block-wrap"><div class="code-block-head"><span>Copyable action prompt</span><button class="code-copy" type="button" aria-label="Copy code" title="Copy code">⧉</button></div><pre class="language-text"><code>Reference this report action: collect source IDs first, then interpret findings into recommendations with owners, acceptance criteria, and rerun steps.
 
 Guide me through the tools, resources, accounts, permissions, source material, and access needed to take this action. Break the work into numbered steps, call out any missing inputs before execution, include safe handling for credentials or confidential data, and finish with verification evidence I can capture.</code></pre></div></details>
 <section class="anchor-links">
 <p><a href="#sources" data-filetype="link">Sources</a> <a href="#highest-impact-tactics" data-filetype="link">Tactics</a> <a href="#page-type-matrix" data-filetype="link">Matrix</a> <a href="#roadmap-template" data-filetype="link">Roadmap</a> <a href="#appendices" data-filetype="link">Appendices</a></p>
 </section>
+<section class="report-keep-with-heading report-chapter-page">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </section> ], start tag match failed [ <section class="report-keep-with-heading report-chapter-page"> ] on line 2508.

(tag-pair)